### PR TITLE
Schema refactor & dotenv config

### PR DESCRIPTION
**Pull request by pjol**
[Ticket 1](https://trello.com/c/CxEjdSb0/30-m-refactor-schema)
[Ticket 2](https://trello.com/c/lhmREhwA/31-untitled)

Added Dotenv functionality and refactored schema to include styles and products models

@justcord10 @LukeLarson2

### CODE REVIEW

File: db/ETL.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const { parse } = require('csv-parse');
 const mongoose = require('mongoose');
-const productsSchema = require('./productsSchemaNew');
+const productsSchema = require('./productsSchema');
+const stylesSchema = require('./stylesSchema');
```
Comment (justcord10) [inline db/ETL.js:4]: broke up into two schemas like ya said you would

File: db/ETL.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 const fs = require('fs');
 
-const url = 'mongodb://localhost:27017/products'
+const url = process.env.MONGO_URL
```
Comment (justcord10) [inline db/ETL.js:7]: good moving into .env, getting set up for deployment

**Review by LukeLarson2 — APPROVED** [review]
Good planning ahead for deployment on having a dynamic connection string

P.S. and theeeeeeeen?

File: db/ETL.js [hunk 3]
```diff
@@ -92,54 +95,54 @@ const populateData = async (path, constructor, page) => {
...
-    return readData(skus, skusConstructor, 5);
+    return readData(stylesModel, skus, skusConstructor, 5);
   })
   .then((res) => {
     delete res;
-    return readData(skus, skusConstructor, 6);
+    return readData(stylesModel, skus, skusConstructor, 6);
   })
   .then(() => {
     console.log('Done!')
```
Comment (justcord10) [inline db/ETL.js:148]: Just broke up for sizing limits

File: db/ETL.js [hunk 4]
```diff
@@ -195,20 +198,14 @@ const stylesConstructor = (data) => {
...
-          original_price: data[4],
-          sale_price: data[3],
-          default_style: data[5],
-        }
-      }
+      style_id: data[0],
+      name: data[2],
+      original_price: data[4],
+      sale_price: data[3],
+      default_style: data[5],
```
Comment (justcord10) [inline db/ETL.js:208]: updated var names from product_id to id and adapted the style model

File: db/ETL.js [hunk 5]
```diff
@@ -269,17 +264,17 @@ const relatedConstructor = (data) => {
...
+module.exports.populatePhotos1 = () => populateData(stylesModel, photos, photosConstructor, 1);
+module.exports.populatePhotos2 = () => populateData(stylesModel, photos, photosConstructor, 2);
+module.exports.populatePhotos3 = () => populateData(stylesModel, photos, photosConstructor, 3);
+module.exports.populateRelated = () => populateData(productsModel, related, relatedConstructor);
+module.exports.populateSkus1 = () => populateData(stylesModel, skus, skusConstructor, 1);
+module.exports.populateSkus2 = () => populateData(stylesModel, skus, skusConstructor, 2);
+module.exports.populateSkus3 = () => populateData(stylesModel, skus, skusConstructor, 3);
+module.exports.populateSkus4 = () => populateData(stylesModel, skus, skusConstructor, 4);
+module.exports.populateSkus5 = () => populateData(stylesModel, skus, skusConstructor, 5);
+module.exports.populateSkus6 = () => populateData(stylesModel, skus, skusConstructor, 6);
```
Comment (justcord10) [inline db/ETL.js:279]: updated for 2 models

File: db/k6.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    { duration: '10s', target: 1000 },
+    { duration: '5s', target: 0 },
+  ],
+}
+
+export default function() {
+  const product = Math.floor(Math.random() * 1000011);
+  http.get(`http://localhost:3000/products/${product}`);
+  sleep(1);
+}
```
Comment (justcord10) [inline db/k6.js:16]: set up k6 testing NICE!

File: db/productsSchema.js [hunk 7]
```diff
@@ -2,25 +2,12 @@ const mongoose = require('mongoose');
...
-    original_price: String,
-    sale_price: String,
-    'default?': Boolean,
-    photos: [{thumbnail_url: String, url: String}],
-    skus: [{
-      sku: String,
-      quantity: String,
-      size: String
-    }]
-  }],
```
Comment (justcord10) [inline db/productsSchema.js:24]: removed styles from this schema

**Review by justcord10 — APPROVED** [review]
Looks good, just left comments through your code saying what i thought it was doing. looks like really good progress/refactoring!

**Pull request merged**